src/effects/VST3/VST3Effect.cpp

```diff
@@ -176,17 +176,11 @@ VST3Effect::VST3Effect(
    VST3::Hosting::ClassInfo effectClassInfo)
       : mModule(std::move(module)), mEffectClassInfo(std::move(effectClassInfo))
 {
-   using namespace Steinberg;
    mWrapper = std::make_unique<VST3Wrapper>(*mModule, mEffectClassInfo.ID());
-   //defaults
-   mWrapper->mSetup.processMode = Vst::kOffline;
-   mWrapper->mSetup.symbolicSampleSize = Vst::kSample32;
-   mWrapper->mSetup.maxSamplesPerBlock = mUserBlockSize;
-   mWrapper->mSetup.sampleRate = 44100.0;
-
-   if(!SetupProcessing(*mWrapper->mEffectComponent, mWrapper->mSetup))
-      //ProcessInitiaize should attempt to change that...
-      mWrapper->mSetup.sampleRate = .0;
+   mWrapper->mSetup.processMode = Steinberg::Vst::kOffline;
+   mWrapper->mSetup.symbolicSampleSize = Steinberg::Vst::kSample32;
+   mWrapper->mSetup.maxSamplesPerBlock = mProcessingBlockSize;
+   mWrapper->mSetup.sampleRate = mProjectRate;
```

Comment:
Just use literal `44100.0` as a default, as before.  Don't use `EffectBase::mProjectRate`.  That is another piece of state in the effect objects I need to eliminate in a future release.

Reply:
mProjectRate value is used to initialize generators, like in another effects, but without explicit sample rate specification

Reply:
Later commits eliminated these lines, so it doesn't matter.

`mSetup` is now, properly, part of Instance state, not the effect, and is initialized with the correct rate.